rust/syntect-plugin/Makefile

```diff
@@ -4,12 +4,26 @@ XDG_CONFIG_HOME ?= $(HOME)/.config
 XI_CONFIG_DIR ?= $(XDG_CONFIG_HOME)/xi
 XI_PLUGIN_DIR ?= $(XI_CONFIG_DIR)/plugins
 
-target/release/xi-syntect-plugin:
+lang_configs = assets/*.toml
+
+out/syntect: $(lang_configs) out/gen_manifest.toml target/release/xi-syntect-plugin
+	mkdir -p out/syntect/bin
+	cp $(lang_configs) out/syntect
+	install target/release/xi-syntect-plugin out/syntect/bin
```

Comment:
Why not use `cp` here as well for the sake of consistency?

Reply:
@eyelash I believe install also sets +x?

Reply:
Shouldn't that already be set by cargo?